scripts/ERC20ForwardersDeployerMainnet.js

```diff
@@ -20,60 +20,89 @@ async function main() {
     const usdtAddress = "0xdAC17F958D2ee523a2206206994597C13D831ec7"; 
     const usdtDecimals = 6;
     
+    //deploy contracts with the final owner address
     const owner = "0xEbdC114433f8119c1367e23A90CBbC7E2D11efBf";
     const ERC20ForwarderProxyAdmin = "0xAD8b61F17c6a53d159baFC67082aC8d0a86e712F";
     const feeReceiver = "0x12349D54Aa10cc5b158D8B5575e11e88cb5Dee09";
+
+    let tx, receipt;
+    let totalGasUsed = 0;
+    var options = { gasPrice: 17000000000, gasLimit: 10000000};
   
     const Forwarder = await hre.ethers.getContractFactory("BiconomyForwarder");
-    const forwarder = await Forwarder.deploy(owner);
+    //todo
+    //fetch gas price and change in in the options
+    //pass options for all transactions
+    const forwarder = await Forwarder.deploy(owner,options);
     await forwarder.deployed();
-  
+    receipt = await forwarder.deployTransaction.wait(confirmations = 2);
     console.log("✅ Biconomy Forwarder deployed at : ", forwarder.address);
-    await forwarder.registerDomainSeparator("Biconomy Forwarder", "1");
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
+     
+    tx = await forwarder.registerDomainSeparator("Biconomy Forwarder", "1");
+    receipt = await tx.wait(confirmations = 2);
+    console.log(`gas used : ${receipt.gasUsed.toNumber()}`);
+    totalGasUsed = totalGasUsed + receipt.gasUsed.toNumber();
   
     const CentralisedFeeManager = await hre.ethers.getContractFactory("CentralisedFeeManager");
     const centralisedFeeManager = await CentralisedFeeManager.deploy(owner, 10000);
```

Comment:
Use the `options` in each deploy statement. Check in other files also.

Reply:
will work on this

Reply:
adding options. Is it sufficient if I do fetch gas price only for mainnet?